src/v/cluster/topic_table.h

```diff
@@ -515,6 +515,10 @@ class topic_table {
 
     updates_t _updates_in_progress;
     model::revision_id _last_applied_revision_id;
+    // Monotonic counter that is bumped for every addition/deletion to topics
+    // map. Unlike other revisions this does not correspond to the command
+    // revision that updated the map.
+    model::revision_id _topics_map_revision{0};
```

Comment:
i think this one should also be incremented whenever the partition assignment set is change i.e. on `create_partition_cmd` handling

Reply:
That makes sense.